client/src/components/dashboard/dashboard-header/index.tsx

```diff
@@ -28,7 +28,9 @@ export const DashboardHeader = () => {
     router.push(routes.home);
   };
 
-  const titleNav = links.find(({ href }) => path.includes(href));
+  const titleNav =
+    links.find((link) => path.includes(link.href)) ||
+    links.flatMap((link) => link.children || []).find((child) => path.includes(child.href));
```

Comment:
`(link) => path.includes(link.href)` can be extracted into a separate function and reused in both those mapping operations